### ISSUE TITLE
Cloud Mode redirects

### ISSUE THREAD

**Issue by atmonshi** [umami-software/umami#1972]
Hi all
cloud mode always redirects to `cloud.umami.is`
found this code :

```
if (process.env.CLOUD_MODE) {
  redirects.push({
    source: '/login',
    destination: CLOUD_URL,
    permanent: false,
  });
}
```

when I tried to disable that code or change the 'CLOUD_URL' to 'http://localhost:3000'
I got **401 Unauthorized** from the endpoint 'api/auth/verify'

any thoughts?

umami v2.1
node v19.9.0
db mysql

**Comment by mikecao** [comment]
That looks like a bug, it should be `process.env.CLOUD_URL`.

**Comment by mikecao** [comment]
Fixed in v2.2.0.
